test/e2e/framework/internal/output/output.go

```diff
@@ -178,3 +179,12 @@ func normalizeLocation(in string) string {
 	out = klogPrefix.ReplaceAllString(out, "<klog> ")
 	return out
 }
+
+var initFunc = regexp.MustCompile(`(init\.+func|glob\.+func)`)
```

Comment:
is this normalzing go1.22 output to match go1.21, or normalizing both to something else? is the glob func _always_ an init func?

does the TestFailureOutput test need to change as well?

Reply:
It's normalizing both to something else (`<init.func>`).

The test continues to use the Go < 1.22 naming with `glob..func`. That could be updated to use `<init.func>` to make it clearer that it is a placeholder - although I probably need to use a placeholder then which isn't matched by the regexp to avoid replacing it again...

Reply:
> is the glob func always an init func

It is in this test case. The pattern doesn't need to be correct for everything, only for functions which appear in the output.